chain/x/oracle/client/common/proof/block_header_merkle_parts.go

```diff
@@ -37,6 +63,7 @@ func (bp *BlockHeaderMerkleParts) encodeToEthFormat() BlockHeaderMerklePartsEthe
 	}
 }
 
+// GetBlockHeaderMerkleParts compacts Tendermint block header to compacted version for optimized gas used.
```

Comment:
GetBlockHeaderMerkleParts converts Tendermint block header struct into BlockHeaderMerkleParts for gas-optimized proof verification.